### PR TITLE
Migrate admin settings to material design icon

### PR DESCRIPTION
**Pull request by GretaD**
~~i dont know how to test this~~
the before its exactly the same but old icons

after
after-admin

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Open `/settings/admin/groupware`

File: src/components/settings/AdminSettings.vue [hunk 1]
```diff
@@ -107,7 +107,7 @@
 			}}
 		</h3>
 		<p>
-			<Button class="config-button" @click="addNew=true">
+			<Button class="config-button new-config" @click="addNew=true">
```
Comment (ChristophWurst) [inline src/components/settings/AdminSettings.vue:110]: bem?

**Review by miaulalala — APPROVED** [review]

**Comment by ChristophWurst** [comment]
CI doesn't pass

**Comment by st3iny** [comment]
> CI doesn't pass

Seems to be a transient issue. I restarted all failed runs.

**Pull request merged**